pkg/bundle/sync.go

```diff
@@ -76,6 +75,14 @@ type bundleData struct {
 	defaultCAPackageStringID string
 }
 
+type jksEncoder struct {
+	password string
+}
+
+type pkcs12Encoder struct {
+	password string
+}
+
```

Comment:
I guess to have structures, global vars and consts as well as imports in one place rather than somewhere in the middle of code lines.
If you think that was bad decision then I can move it back.

Reply:
I think we should avoid moving it in this PR. I personally prefer to have the structs defined together with the methods having the structs as receiver. But the main point here is to not move things just out of personal preference.